deploy/Dockerfile

```diff
@@ -1,11 +1,13 @@
-# sha256 as of 2021-11-09
-FROM python:3.9-slim-bullseye@sha256:408de0cf1a057f5501ee6642ad24a4762738f63bacf09fb4c8d861669260b01e AS sphinx
+# sha256 as of 2023-01-17
+FROM python:3.9-slim-bullseye@sha256:8b0502fe3a8ae9d26567f66e217223595ff6abe901cba64fd8b1b58176311b27 AS sphinx
 
 ARG GIT_BRANCH=main
 RUN apt-get -q update && apt-get -qy upgrade && apt-get -qy install git make latexmk texlive-latex-extra
 COPY ./ .
-RUN pip install -r requirements/requirements.txt
-RUN deploy/build $GIT_BRANCH
+RUN pip install poetry==1.4.0
+RUN poetry install
+# TODO: Once the latest stable tag uses poetry, we can drop the `poetry run` prefix
```

Comment:
I don't understand this comment, why are the `poetry run` in the Makefile not sufficient?

Reply:
This repo uses a more complex versioning strategy than `redmine_docs`. Here, a git checkout of the most recent version is performed to render the stable version. That version doesn't have the required `Makefile` changes yet which wrap the `sphinx-build` invocations, so it'll error out during the build process

Reply:
OK, I re-read `deploy/build` and now I remember how it works. Ack!